### PR TITLE
Test rustc_boehm integration as part of bors

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

**Comment by jacob-hughes** [comment]
bors try

**Comment by jacob-hughes** [comment]
bors try

**Comment by jacob-hughes** [comment]
bors try

**Comment by jacob-hughes** [comment]
bors try

**Comment by ltratt** [comment]
Is this ready for review or ...?

**Comment by jacob-hughes** [comment]
Yes sorry I forgot to comment on the this.

**Comment by ltratt** [comment]
Will we cache rustc_boehm anywhere, or is that a bit difficult to set up?

**Comment by jacob-hughes** [comment]
No. That would be ideal but I'm not sure how we could set this up. I suppose we could build a new `rustc_boehm` bin each time a PR lands on that repo, but it would be a bit fiddly I think.

File: .buildbot.sh [hunk 1]
```diff
@@ -24,3 +24,18 @@ cargo run --release -- --cp SOM/Smalltalk SOM/TestSuite/TestHarness.som
 
 rustup toolchain install nightly --allow-downgrade --component rustfmt
 cargo +nightly fmt --all -- --check
+
+# Build rustc_boehm
+git clone https://github.com/softdevteam/rustc_boehm
+mkdir -p rustc_boehm/build/rustc_boehm
+(cd rustc_boehm && ./x.py build --config ../.buildbot.config.toml)
```
Comment (ltratt) [inline .buildbot.sh:31]: I'm not sure the brackets are doing anything here?

Comment (ltratt) [reply .buildbot.sh:31]: Oh, is it to make the `cd` local?

Comment (jacob-hughes) [reply .buildbot.sh:31]: Yes that's right

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Comment by jacob-hughes** [comment]
This should be merge-able again once https://github.com/softdevteam/rboehm/pull/26 has landed.

**Pull request merged**